packages/augur-sdk/src/state/getter/Markets.ts

```diff
@@ -559,8 +559,13 @@ export class Markets {
       if (params.includeInvalidMarkets !== true && market.invalidFilter) return false;
       // Liquidity filtering
       if (params.maxLiquiditySpread) {
-        // TODO hasRecentlyDepletedLiquidity == true on ZeroPercent spread
-        if (params.maxLiquiditySpread !== MaxLiquiditySpread.OneHundredPercent) {
+        if (params.maxLiquiditySpread === MaxLiquiditySpread.ZeroPercent) {
+          // return hasRecentlyDepletedLiquidity on ZeroPercent spread
+          if (!market.hasRecentlyDepletedLiquidity) {
```

Comment:
I don't think this is right. hasRecentlyDepletedLiquidity needs to also consider when the market last passed the liquidity check. That's all that commented out code getting deleted in this PR. It was implemented in a very bad way originally and the issue to restore it correctly has been languishing.